### PR TITLE
Fixup python3 path

### PR DESCRIPTION
**Pull request by ivan-c**
- Update version-specific python3 paths to bookworm release of python3
- fixes #4342

### CODE REVIEW

File: debian/portal.triggers [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Register interest in Python interpreter changes (Python 2 for now); and
```
Comment (pbugni) [inline debian/portal.triggers:1]: dated comment (Python 2), but not urgent.

**Review by pbugni — APPROVED** [review]
would rather see a variable used for python version, but i assume this is not easy/available, or you would have done so.

File: debian/portal.triggers (hunk 1 above)
Comment (ivan-c) [inline debian/portal.triggers:1]:
```suggestion
# Register interest in Python interpreter changes (Python 3 for now); and
```

Comment (ivan-c) [reply debian/portal.triggers]: Thanks! fixed in 8be5066

**Pull request merged**